uniffi_bindgen/src/templates/CallbackInterfaceTemplate.rs

```diff
@@ -0,0 +1,102 @@
+{#
+// For each callback interface in the UDL, we assume the caller has provided a corresponding
+// Rust `trait` with the declared methods. We provide the traits for sending it across the FFI.
+// If the caller's trait does not match the shape and types declared in the UDL then the rust
+// compiler will complain with a type error.
+// 
+// The generated proxy will implement `Drop`, `Send` and `Debug`.
```

Comment:
The comment here doesn't explain what the "generated proxy" is, it may be worth a sentence or two to do so (otherwise I don't know what the significance of it implementing these traits is)